### PR TITLE
VAN-82545: add fake cwc routes

### PR DESCRIPTION
**Pull request by tirish**
- remove bodyparser (it is deprecated)
- use `express.raw()` to handle XML content

For now, always respond with an empty 200 response.
Also, for now, have the office list for CWC match the test list and the SCWC list be a some subset. Will likely tweak these in the future.

### CODE REVIEW

File: lib/routes/cwc.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+function constructPayload(req, type){
+    // using the express.raw() middleware sets the req.body to a buffer that we can convert back into a string
+    var xmlString = req.body.toString();
+    return xml.parseStringPromise(xmlString).then(function(parsed){
+        var email = _.get(parsed, 'CWC.Constituent[0].Email[0]') || 'no-email';
+        return {
+            cwcType: type,
+            email: email,
+            hasApiKey: req.query.apikey ? 'yes' : 'no',
```
Comment (drewsposeep) [inline lib/routes/cwc.js:19]: do you want us to send a stub API key to jane casey for consistency?

**Review by drewsposeep — APPROVED** [review]

File: lib/routes/cwc.js (hunk 1 above)
Comment (tirish) [reply lib/routes/cwc.js:19]: we should send something, probably shouldnt be the live key

**Comment by tirish** [comment]
Going to go ahead and merge/deploy this. Can revert or tweak if needed

**Comment by tirish** [comment]
There was one issue to fix after merging this: https://github.com/NGPVAN/jane-caseys-congress-form/commit/5541af5f52aac6b933b027bcd0f882e18d2cd5fb

**Pull request merged**